public/models/interfaces.ts

```diff
@@ -234,14 +234,23 @@ export type AnomalyData = {
   plotTime?: number;
   entity?: EntityData[];
   features?: { [key: string]: FeatureAggregationData };
-  aggInterval?: string;
+  contributions?: { [key: string]: FeatureContributionData };
+  aggInterval?: string; 
 };
 
 export type FeatureAggregationData = {
   data: number;
+  name?: string;
```

Comment:
confused on this name property, did we not have this before at all?

Reply:
no, we are passing feature id around, not feature name